### PR TITLE
[WIP] adds obol icons to the obol-ui and info cards

### PR DESCRIPTION
**Pull request by f1lander**
## description

This PR adds obol icons as components and info cards

### CODE REVIEW

File: components/molecules/InfoCard/InfoCard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Link } from "../../atoms";
+import { IconArrowForward } from "../../icons";
+
+import { InfoCard, InfoCardProps } from "./InfoCard";
+
+export default {
+  title: "Design System/Molecules/InfoCard",
```
Comment (xenowits) [inline components/molecules/InfoCard/InfoCard.stories.tsx:8]: nit: 
```suggestion
  title: "DesignSystem/Molecules/InfoCard",
```

**Review by xenowits — APPROVED** [review]
Looks good!
A small question though: Are the svg icon files inside `components/icons/static-icons` generated manually?

**Comment by f1lander** [comment]
@xenowits correct, I could do via script consuming figma Api but it will not a good approach since is something that will not change often.

I can explain you in our next meeting why I create parent component call SvgIcon to slider copy the svg content

**Comment by f1lander** [comment]
### Update

image

File: components/molecules/InfoCard/InfoCard.stories.tsx (hunk 1 above)
Comment (f1lander) [reply components/molecules/InfoCard/InfoCard.stories.tsx:8]: @xenowits this is ok with the space, this correspond to the folder structure 
image

**Pull request merged**